code/frameworks/ember/src/client/preview/render.ts

```diff
@@ -2,7 +2,7 @@ import { global } from '@storybook/global';
 import { dedent } from 'ts-dedent';
 import type { RenderContext } from '@storybook/types';
 // @ts-expect-error (Converted from ts-ignore)
```

Comment:
I guess we can remove this line this types for `@glimmer` should be defined

Reply:
Done